app/brave_generated_resources.grd

```diff
@@ -396,6 +396,10 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
       <message name="IDS_SETTINGS_BRAVE_ON_EXIT" desc="Clear Browsing Data dialog On exit tab label">
         On exit
       </message>
+      <!-- In-page Translation -->
+      <message name="IDS_BRAVE_TRANSLATE_BUBBLE_BEFORE_TRANSLATE_TITLE" desc="Title text for the translate bubble when asking to translate a page.">
+        Translate this page using Microsoft Cognitive Services? Texts in this page will be sent to Microsoft server for translation.
```

Comment:
[minor] this sounds like it should either say "sent to Microsoft" or "sent to Microsoft's servers"

Reply:
@diracdeltas How about this?
<img width="342" alt="Screen Shot 2019-03-20 at 1 05 06 PM" src="https://user-images.githubusercontent.com/4730197/54715665-30658e80-4b11-11e9-869e-6eb6e1484d3e.png">